sensor_msgs/CameraInfo cam_info
sensor_msgs/Image color_img
sensor_msgs/Image depth_img
std_msgs/String target_name
bool debug_mode
bool ground
---
uint32[] encoded_masks
bool success
